src/components/ThreeDEditor.jsx

```diff
@@ -119,8 +121,21 @@ export class ThreeDEditor extends React.Component {
         });
     };
 
+    // eslint-disable-next-line class-methods-use-this
     handleCellRepetitionsChange(e) {
-        this.handleSetSetting({ repetitions: parseFloat($(e.target).val()) });
+        const axis = e.target.id
```

Comment:
If `e.target.id` is the key in `viewerSettings` then this basically simplifies to `this.handleSettings({ [e.target.id]: floatVal })` right?